stdlib/src/builtin/sort.mojo

```diff
@@ -210,15 +215,16 @@ fn _quicksort[
 # ===----------------------------------------------------------------------===#
 # partition
 # ===----------------------------------------------------------------------===#
+@always_inline
```

Comment:
+1 on always inlining the hidden methods, but I'm concerned about potential code bloat with decorating the public APIs. Did you benchmark the effects of these, and if so, did you find a perf reason for doing this?

Reply:
Totally agree with you. The implication of code bloat slipped my mind. I removed the always inline decorators from public API.